crates/starknet-server/src/api/json_rpc/endpoints.rs

```diff
@@ -1,3 +1,5 @@
+use server::rpc_core::error::RpcError;
+use starknet_types::felt::Felt;
```

Comment:
this can be reshaped into

`use starknet_types::{felt::Felt, starknet_api::block::BlockNumber};`

This is preferred way it allows to save space and a general approach of imports in rust

Reply:
Interesting, it seems our formatter prefers it the way it is (i.e. separate lines). So I'll just resolve this conversation.

Reply:
The formatter doesn't automatically create such import so its up to developers. Otherwise our import list will get too big

Reply:
Not sure what you mean, I tried applying the change you proposed, and after running `./scripts/format.sh`, it was back to as it is now.

Reply:
Our CI/CD checks that all formatter rules are obeyed and I'm ok with listening to the formatter.

Reply:
Will create separate for 

`imports_granularity = "Module"` -> i`mports_granularity = "Crate"`